app/src/main/kotlin/com/simplemobiletools/dialer/activities/CallActivity.kt

```diff
@@ -98,7 +98,22 @@ class CallActivity : SimpleActivity() {
     }
 
     private fun initButtons() {
-        handleSwipe()
+        if (!config.disableSwipeToAnswer) {
```

Comment:
lets flip this check, "if (config.disableSwipeToAnswer) {" is easier to read if we are working with both true and false cases